tests/zfs-tests/cmd/xattrtest/xattrtest.c

```diff
@@ -630,6 +631,7 @@ unlink_files(void)
 		rc = unlink(file);
 		if ((rc == -1) && (errno != ENOENT)) {
 			ERROR("Error %d: unlink(%s)\n", errno, file);
+			free(file);
```

Comment:
Never mind, you're right.  I got this hunk confused with the `file` variable in `drop_caches()` from the previous hunk.